openfecli/commands/input.yaml

```diff
@@ -0,0 +1,43 @@
+# things behind a "method" key have to match up to a known thing in openfe
+
+network:
+  method: plan_radial_network
+  scorer: default_lomap_score
+
+# Question:
+# use settings here, to match Protocol
+
+mappers:
+  - method: LomapAtomMapper
+    settings:
+      timeout: 60
+  - method: LomapAtomMapper
+    settings:
+      timeout: 120
+
+# OR not nesting settings, anything without the "method" key gets put into kwargs
```

Comment:
Either one works from a JSON Schema standpoint. I'd vote to start by requiring it to be a list and add handling for non-list later, just as a matter of do the simple thing first.